src/containers/WorkspaceView.css

```diff
@@ -0,0 +1,14 @@
+.link-button {
```

Comment:
This .link-button class allows us to make a `<button onClick={...} className="link-button">...</button>` look like a hyperlink.